service/service.go

```diff
@@ -702,10 +719,12 @@ func (s *service) getTransportProtocolFromEnv() string {
 			break
 		case "ISCSI":
 			break
+		case "NVMETCP":
```

Comment:
Is this updated in helm-chart or secret, from anywhere this is taken and set to environment var that these are the allowed values as this seems to be case sensitive here?